tools/devworkspace-generator/tests/main.spec.ts

```diff
@@ -275,7 +275,7 @@ describe('Test Main with stubs', () => {
       } catch (e) {
         message = e.message;
       }
-      expect(message).toEqual('missing editorPath or editorEntry or editorContent');
+      expect(message).toEqual('missing editorPath or editorEntry or editorContent or editorUrl');
```

Comment:
The Jest is unhappy because of uncovered lines:

<img width="734" alt="image" src="https://user-images.githubusercontent.com/5676062/222164263-1a54e3bf-d54a-4106-8e5c-b0a65dd4ca27.png">

working on it, will update the PR soon...